internal/services/gateway/action/run.go

```diff
@@ -167,7 +167,7 @@ type DeleteLogsRequest struct {
 }
 
 func (h *ActionHandler) DeleteLogs(ctx context.Context, req *DeleteLogsRequest) error {
-	canDoRunActions, groupID, err := h.CanAuthUserDoRunActions(ctx, req.GroupType, req.Ref)
+	canDoRunActions, groupID, err := h.CanAuthUserDeleteLogs(ctx, req.GroupType, req.Ref)
```

Comment:
Why have you split CanAuthUserDoRunActions in two functions if they do the same thing?